screens/PostFeed.tsx

```diff
@@ -1,11 +1,14 @@
 import * as React from 'react';
-import { FlatList, View } from 'react-native';
+import { FlatList, View, Text } from 'react-native';
 import { RootTabScreenProps } from '../types';
 import { Post } from '../components/Post/Post';
 import { StyleSheet } from 'react-native';
+import axios from 'axios';
 
 // TODO: Fetch posts from api endpoint
```

Comment:
Gjort nu va? :DD

Reply:
Kommentaren tillhör alltså rad 8, inte 6

Reply:
that is true